README.md

```diff
@@ -12,10 +12,58 @@ TO DO
 
 TO DO
 
-### Datasets
+## Datasets
 
 TO
 
+## EleutherAI cluster
+
+### Using a cluster
+
+If you already have a EleutherAI cluster setup for you, test to see if the cluster is working:
+    
+1. Copy data from cold storage to the shared mount: `cp -R /mnt/ssd-0/megatron-3d/data /mnt/ssd-cluster/data`
```

Comment:
I don't know if we should be putting commands that are specific to eleuther pods in the general readme - maybe a different branch for this?

Reply:
We already have EleutherAI cluster specific stuff in this repo... the kubernetes scripts and dockerimage ect. We could add this to a readme in the kubernetes folder?